great_expectations/rule_based_profiler/estimators/exact_numeric_range_estimator.py

```diff
@@ -56,8 +60,8 @@ def _get_numeric_range_estimate(
             parse_strings_as_datetimes=True,
             fuzzy=False,
         )
-        min_value: MetricValue = np.amin(a=metric_values_converted)
-        max_value: MetricValue = np.amax(a=metric_values_converted)
+        min_value = np.amin(a=metric_values_converted)
```

Comment:
@Kilo59 How come removing the `MetricValue` type hint is proposed here?  Thanks.

Reply:
Because `build_numeric_range_estimation_result()` expects numbers for `min_value` and `max_value` and `MetricValue` is a much broader union of types.

```
great_expectations/rule_based_profiler/estimators/exact_numeric_range_estimator.py:69: error: Argument "min_value" to "build_numeric_range_estimation_result" has incompatible type "Union[Any, List[Any], Set[Any], Tuple[Any, ...], Any, Any, ndarray[Any, Any], int, str, float, bool, Dict[str, Any]]"; expected "Number"  [arg-type]
```

Reply:
@Kilo59 This warning is not showing up in my `PyCharm`, which is strange.  Regardless, what if we change the type hint to `Number`?  Thanks.

Reply:
Done